### ISSUE TITLE
How to conduct the video-text matching loss mentioned in the paper for ViCLIP during text-video retrieval fine-tuning

### ISSUE THREAD

**Issue by jpWang** [OpenGVLab/InternVideo#111]
Hi, thanks for your great work！
In the InternVid paper, it said that the video-text matching loss is also added for ViCLIP during text-video retrieval fine-tuning.
![image](https://github.com/OpenGVLab/InternVideo/assets/52776462/1197f52f-3ca0-4adc-9100-63ac2b8aeb9b)
What confuses me is how exactly this loss is done? Isn't the video-text matching loss usually achieved by adding new binary classification head upon cross-modal features? What should this be done for ViCLIP?
Looking forward to your reply~
@shepnerd

**Comment by leexinhao** [comment]
This was a clerical error and we only used contrastive loss during the full-finetuning setting.
